ppr-ui/src/components/search/SearchBar.vue

```diff
@@ -14,149 +14,175 @@
-              filled
-              :hint="searchHint"
-              :hide-details="hideDetails"
-              persistent-hint
-              :placeholder="selectedSearchType ? selectedSearchType.textLabel: 'Select a category first'"
-              v-model="searchValue"
-              @keypress.enter="searchCheck()"
-            />
-          </template>
-          <v-row v-for="(line, index) in searchPopUp" :key="index" class="pt-2 pl-3">
-            {{ line }}
-          </v-row>
-        </v-tooltip>
-        <auto-complete :searchValue="autoCompleteSearchValue"
-                       :setAutoCompleteIsActive="autoCompleteIsActive"
-                       v-click-outside="setCloseAutoComplete"
-                       @search-value="setSearchValue"
-                       @hide-details="setHideDetails">
-        </auto-complete>
-      </v-col>
-      <v-col v-else cols="7" class="pl-3">
-        <v-row no-gutters>
-          <v-col cols="4">
-            <v-text-field
-              id="first-name-field"
-              :class="[$style[wrapClass]]"
-              autocomplete="off"
-              :error-messages="searchMessageFirst ? searchMessageFirst : ''"
-              filled
-              :hint="searchHintFirst"
-              persistent-hint
-              :placeholder="optionFirst"
-              v-model="searchValueFirst"
-              @keypress.enter="searchCheck()"
-          />
+    <v-row>
+      <v-col class="col-xl py-0">
+        <v-row>
+          <v-col class="search-info py-0">
+            Select a search category and then enter a criteria to search.
           </v-col>
-          <v-col cols="4" class="pl-3">
-            <v-text-field
-              id="second-name-field"
-              autocomplete="off"
-              :error-messages="searchMessageSecond ? searchMessageSecond : ''"
-              filled
-              :hint="searchHintSecond"
-              persistent-hint
-              placeholder="Middle Name (Optional)"
-              v-model="searchValueSecond"
-              @keypress.enter="searchCheck()"
+          <v-col align-self="end" cols="3" class="py-0">
+            <folio-number
+              :defaultFolioNumber="folioNumber"
+              @folio-number="updateFolioNumber"
+              @folio-error="folioError = $event"
             />
           </v-col>
-          <v-col cols="4" class="pl-3">
-            <v-text-field
-              id="last-name-field"
-              autocomplete="off"
-              :error-messages="searchMessageLast ? searchMessageLast : ''"
-              filled
-              :hint="searchHintLast"
-              persistent-hint
-              placeholder="Last Name"
-              v-model="searchValueLast"
-              @keypress.enter="searchCheck()"
+        </v-row>
+        <div v-if="typeOfSearch">
+          <div class="pt-0 pb-1">
+            <div v-html="typeOfSearch" class="font-weight-bold search-title"></div>
+          </div>
+        </div>
```

Comment:
Not so sure about these layers of divs, are they really all required? Why not 1 div? or 2 at most perhaps?

Reply:
Right, there were rows/cols before. Will simplify.